### ISSUE TITLE
Proper documentation on gitbooks

### ISSUE THREAD

**Issue by danielearwicker** [danielearwicker/immuto#6]
This is underway but will take a while...

**Comment by basarat** [comment]
Let me know if you have any questions. I went with gitbook 3.0 for https://github.com/alm-tools/alm as it supported _in repo_ docs more easily. Also redux uses 3.0.0 for the same reason : https://github.com/reactjs/redux/blob/master/book.json :rose:

**Comment by basarat** [comment]
Also are you thinking of making `import * as I from "immuto"` defacto? If so maybe the logo should have capital `I`?

PS nice logo :rose:

**Comment by danielearwicker** [comment]
Yep will go with gitbooks. I prefer importing the names I need instead of `* as I`, but I use the latter to work around that extra error we get when generating declarations. Thanks for all your help btw!
